.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"
```

Comment:
Currently blocked by `https://github.com/travis-ci/travis-ci/issues/9552`.

Reply:
See also https://github.com/travis-ci/travis-ci/issues/9815

Reply:
Maybe we can start installing Miniconda and create different versioned Python environments for each matrix element?

Reply:
Looks there are workarounds for it, for instance, https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-402045581 . I don't think this is super urgent and we could just wait for its support though.